packages/react/src/components/form/initial/controls.tsx

```diff
@@ -46,11 +50,13 @@ export const FACTOR_LABEL_MAP: Record<
 export const TAB_NAME = {
   email: "email",
   phone: "phone",
+  username: "username",
 };
 
 export const tabIDByHandle: Record<HandleType, string> = {
   phone_number: TAB_NAME.phone,
   email_address: TAB_NAME.email,
+  username: TAB_NAME.username,
```

Comment:
Same concern as for dynamic flow
> Does it make sense to stick to tabs here? Won't this be a little bit too cramped?

Reply:
https://github.com/slashid/javascript/pull/248#discussion_r1590882155